lib/pagseguro/subscription_payment_order.rb

```diff
@@ -21,14 +21,19 @@ class SubscriptionPaymentOrder
     attr_accessor :last_event_date
     attr_accessor :scheduling_date
     attr_reader :discount
-    attr_reader :transaction
 
     def discount=(discount)
       @discount = ensure_type(PagSeguro::SubscriptionDiscount, discount)
     end
 
-    def transaction=(transaction)
-      @transaction = ensure_type(PagSeguro::SubscriptionTransaction, transaction)
+    def transactions
+      @transactions ||= SubscriptionTransactions.new
+    end
+
+    def transactions=(attrs)
+      attrs.map do |params|
```

Comment:
Well… if you are using `<<` above you don't need `map`, just a `each` works :ok_hand: